shared/constants/network.ts

```diff
@@ -99,8 +99,6 @@ export type NetworkTypes = (typeof NETWORK_TYPES)[keyof typeof NETWORK_TYPES];
 export const NETWORK_NAMES = {
   HOMESTEAD: 'homestead',
 };
-
-export const CHAIN_SPEC_URL = 'https://chainid.network/chains.json';
```

Comment:
moved to `shared/lib/network-utils.ts`, and is no longer exported, as we don't want accidental use of this URL without checking the `useSafeChainsListValidation` user security setting.